orocos_kdl/src/trajectory.cpp

```diff
@@ -62,8 +62,8 @@ Trajectory* Trajectory::Read(std::istream& is) {
 	Eat(is,'[');
 	if (strcmp(storage,"SEGMENT")==0) {
 		IOTrace("SEGMENT");
-		auto_ptr<Path>      geom(    Path::Read(is)       );
-		auto_ptr<VelocityProfile> motprof( VelocityProfile::Read(is)  );
+		unique_ptr<Path>      geom(    Path::Read(is)       );
```

Comment:
The comment above can be updated too.

https://github.com/ros2/orocos_kinematics_dynamics/blob/d842a5ccbfe7579427ebfe7be798c5f45a427b6c/orocos_kdl/src/trajectory.cpp#L58